### DISCUSSION TITLE
Webhook Event History

### DISCUSSION THREAD

**Discussion by skwaugh** [supabase/supabase#15103] [Questions] [answered]
Is it possible to view a history of webhook events, and the response received?

I know there is `supabase_functions.hooks`, which is a partial solution for me - it tells me that it fired, but it doesn't indicate if it was succesful, nor if it ever ran (there's a `created_at`, but no `finished_at`). 

I thought that the webhooks would queue up in `net.http_request_queue`, and the responses would populate in `net._http_response`, but both of those tables are empty.

To access `supabase_functions.hooks`, I did have to grant `postgres` additional permissions.  Is a similar GRANT required to view the data in `net`?

According to Supabase AI, there is a webhook event history, but I didn't see that...

<img width="773" alt="supamkt justifi Supabase 2023-06-15 at 5 17 28 PM" src="https://github.com/supabase/supabase/assets/4550411/4e3313a4-aad0-495f-b26d-416f1d872187">

**Comment by notKeion** [comment]
any update on this?

**Comment by mansueli** [comment, marked as answer]
Here's how you can check the events fired:
```sql
select * from supabase_functions.hooks order by request_id desc limit 25;
```

It doesn't store the response. You can deploy your own webhooks and get the response in pg_net if you want.
